diff --git a/node_modules/react-native-fast-tflite/android/build.gradle b/node_modules/react-native-fast-tflite/android/build.gradle
index 0083618..31b057b 100644
--- a/node_modules/react-native-fast-tflite/android/build.gradle
+++ b/node_modules/react-native-fast-tflite/android/build.gradle
@@ -123,7 +123,7 @@ dependencies {
   // For < 0.71, this will be from the local maven repo
   // For > 0.71, this will be replaced by `com.facebook.react:react-android:$version` by react gradle plugin
   //noinspection GradleDynamicVersion
-  implementation "com.facebook.react:react-native:+"
+  implementation("com.facebook.react:react-android")
 
   // Tensorflow Lite .aar (includes C API via prefabs)
   implementation "com.google.ai.edge.litert:litert:1.0.1"
